core/server/worker/src/main/java/alluxio/worker/block/BlockSyncMasterGroup.java

```diff
@@ -50,8 +50,8 @@ public class BlockSyncMasterGroup implements Closeable {
   private static BlockMasterClientFactory sBlockMasterClientFactory
       = new BlockMasterClientFactory();
 
-  private static final long WORKER_MASTER_CONNECT_RETRY_TIMEOUT =
-      Configuration.getMs(PropertyKey.WORKER_MASTER_CONNECT_RETRY_TIMEOUT);
+  private static final long WORKER_MASTER_CONNECT_RETRY_TIMEOUT = Math.min(
+      Configuration.getMs(PropertyKey.WORKER_MASTER_CONNECT_RETRY_TIMEOUT), Integer.MAX_VALUE);
```

Comment:
It would be better if you let user know the user input value is not used, but instead of the `Integer.MAX_VALUE`

Reply:
Thanks for reviewing the pull request, let's discuss this on the next community meeting.